CMakeLists.txt

```diff
@@ -461,3 +461,30 @@ configure_file(${CMAKE_SOURCE_DIR}/include/freetds/version.h.in ${CMAKE_BINARY_D
 if(NOT EXISTS "${CMAKE_SOURCE_DIR}/configure")
 	configure_file(${CMAKE_SOURCE_DIR}/src/odbc/version.rc.in ${CMAKE_SOURCE_DIR}/src/odbc/version.rc)
 endif(NOT EXISTS "${CMAKE_SOURCE_DIR}/configure")
+
+
+if(WIN32)
+set(FREETDS_PUBLIC_INCLUDE 
+		${CMAKE_SOURCE_DIR}/include/bkpublic.h
+        ${CMAKE_SOURCE_DIR}/include/cspublic.h
+        ${CMAKE_SOURCE_DIR}/include/cstypes.h
+        ${CMAKE_SOURCE_DIR}/include/ctpublic.h
+		${CMAKE_SOURCE_DIR}/include/sqldb.h
+		${CMAKE_SOURCE_DIR}/include/sqlfront.h
+        ${CMAKE_SOURCE_DIR}/include/sybdb.h
+		${CMAKE_SOURCE_DIR}/include/sybfront.h 
+        ${CMAKE_SOURCE_DIR}/include/syberror.h
+		${CMAKE_SOURCE_DIR}/include/odbcss.h
+		${CMAKE_BINARY_DIR}/include/tds_sysdep_public.h)
+
+set(FREETDS_PUBLIC_INCLUDE_2 
+		${CMAKE_BINARY_DIR}/include/freetds/version.h)
```

Comment:
why you want to install that file?

Reply:
I would like verify if freetds version is greater than a specific value.
I can use TDS_VERSION_MAJOR, TDS_VERSION_MINOR,... to assert the version when code is compiled.

Regarding tds_sysdep_public.h I saw it in Makefile.am and I added it. But as I said in previous comment, I only use ctlib so I don't know if all headers included and libraries are necesary.